JSDemos/Demos/ActionSheet/Basics/description.md

```diff
@@ -1 +1,17 @@
-The ActionSheet component is&nbsp;a&nbsp;sheet containing a&nbsp;set of&nbsp;buttons located one under the other. These buttons usually represent several choices relating to&nbsp;a&nbsp;single task. This demo shows how to&nbsp;create a&nbsp;simple ActionSheet and handle item clicks.
\ No newline at end of file
+The ActionSheet component is a popup sheet that contains a set of buttons. These buttons allow users to perform custom actions (call, send, delete, edit, etc). This demo demonstrates how to create a simple ActionSheet and handle the item clicks.
+
+## Specify Controls and Title
+
+ActionSheet can contain multiple buttons. Use either the [dataSource](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/#dataSource) or [items](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/items/) properties to add buttons and configure them. For each button you can specify a [text](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/items/#text), [type](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/items/#type), [icon](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/items/#icon), [template](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/items/#template) and other properties. In this demo, the ActionSheet buttons are configured in a simple array assigned to the **dataSource** property. Each button includes the **text** property only.
+
+If you want to display a title, assign the needed text to the [title](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/#title) property.
+
+## Handle the Item Clicks
+
+The component can perform a custom action when the user clicks the button. Use the [onItemClick](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/#onItemClick) property to handle a custom item click.
+
+You can also handle the click on the default **Cancel** button. For this, assign a custom handler to the [onCancelClick](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/#onCancelClick) property. 
+
+In this demo, the ActionSheet component displays notification with a button name, when you click the button. 
+
+
```

Comment:
- "The component can perform a custom action when the user clicks the button." - We already told about it in the intro.
- When there are many short paragraphs, text looks choppy. Strive to keep the balance.

```suggestion
You can assign a function to the [onItemClick](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/#onItemClick) property to handle button clicks. Use the `e.itemData` field within the function to determine which button was clicked. You can also use the [onCancelClick](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/#onCancelClick) property handle a click on the built-in **Cancel** button. In this demo, the ActionSheet component displays a notification with a button name when you click a button. 
```